<launch>
  <node name="image_view" pkg="image_view" type="image_view" output="screen">
<!--
    <remap from="image" to="image_raw" />
-->


    <remap from="image" to="/AR/camera_image" />

  </node>
</launch>
